### DISCUSSION TITLE
About Resize in Project-MONAI / tutorials/detectuib

### DISCUSSION THREAD

**Discussion by Dana136484** [Project-MONAI/tutorials#1602] [Q&A] [answered]
I need to train my spine dataset to obtain the boxes for each vertebra.The method of Resized was used to process CT before training.When I was training,the method of Resized reported an error.How to solve it?Thank you!The error is as follows:
![problem](https://github.com/Project-MONAI/tutorials/assets/126674286/00e5efe3-c6d3-4d48-8017-7704519e4fd1)

**Comment by KumoLiu** [comment]
Hi @Dana136484, please ensure your data shape is correct. From the error message, your data is 4D, perhaps you should remove one dim.
Thanks!

**Reply by Dana136484** [reply]
Hi,this problem has been resolved.I would like to ask how to change the bounding box of labels of datasets after resizing.Thanks.

**Reply by KumoLiu** [reply, marked as answer]
Do you mean to resize the bounding box? You can try to use `ResizeBox`.
https://github.com/Project-MONAI/MONAI/blob/8fa6931b14ba9617a595fff1d396ac44cc82e207/monai/apps/detection/transforms/array.py#L267
Thanks!

**Reply by Dana136484** [reply]
Yes.Thank you very much!
